### PR TITLE
Allow CAP to browse production clusters

### PR DESCRIPTION
**Pull request by jim-minter**
Allows CAP to browse production clusters

### CODE REVIEW

File: cap/pkg/clusters/development.go [hunk 1]
```diff
@@ -2,60 +2,56 @@ package clusters
...
-	kapi "k8s.io/client-go/tools/clientcmd/api"
-	"k8s.io/client-go/tools/clientcmd/api/latest"
-	"k8s.io/client-go/tools/clientcmd/api/v1"
-
 	"github.com/openshift/openshift-azure/pkg/api"
+	"github.com/sirupsen/logrus"
 )
 
-var excludeResourceGroups = regexp.MustCompile(`^(ci-op-.*|sre)$`)
+var excludeResourceGroups = regexp.MustCompile(`^(ci-op-.*|[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12})$`)
```
Comment (kwoodson) [inline cap/pkg/clusters/development.go]: Do we also want to add a list of strings like, "dev-.*|test-*" and starting using that prefix for our team so our development clusters do not show up?

**Comment by kwoodson** [comment]
Lot of code here but afaics, /lgtm

File: cap/pkg/clusters/development.go (hunk 1 above)
Comment (jim-minter) [reply cap/pkg/clusters/development.go]: this is the list of development clusters, not production ones.  We ignore ci clusters and UUIDs (these are production clusters built in the development subscription)

**Comment by jim-minter** [comment]
@kwoodson please feel free to lgtm - will be building on this

**Pull request merged**